README.md

```diff
@@ -148,7 +148,7 @@ xdp-pass-all-nodes   pass              {}             0          {"primarynodein
 
 ## API Types Overview
 
-Refer to [api-spec.md](./api-spec.md) for a detailed description of all the bpfman Kubernetes API types.
+Refer to [api-spec.md](https://github.com/bpfman/bpfman/blob/main/docs/developer-guide/api-spec.md) for a detailed description of all the bpfman Kubernetes API types.
```

Comment:
I think this is the link you want: https://bpfman.io/main/developer-guide/api-spec/
When I clicked on the link you provided, this was the content:
```
API Specification
--8<-- "./bpfman-operator/apidocs.html"
```